src/components/KeyboardShortcuts/index.tsx

```diff
@@ -145,6 +155,10 @@ export class KeyboardShortcutsBase extends React.Component<Props> {
             log.warn('Cannot navigate to previous change without diff loaded');
           }
           break;
+        case 'h':
+          dispatch(fullscreenGridActions.toggleMainSidePanel());
+          dispatch(fullscreenGridActions.toggleAltSidePanel());
```

Comment:
For the first iteration, I think `h` should only toggle the file tree panel, not the code overview panel. The old review tool did not toggle the code overview with `h` and I can see how one would want to keep it open even while focusing on the code -- the code overview is really helpful for large files. I say we launch it with that behavior and get some feedback from reviewers. 

Additionally, if you have one side panel collapsed before pressing `h`, the two panels switch off/on which doesn't seem like a helpful feature.

Reply:
Ah, yep, why not. The code overview did not exist in the old tool, so this keyboard shortcut was actually displaying the code using the full width, hence both side panels collapsed, but we can start with only hiding the left panel.